admin-src/DiagnoseResponse/index.js

```diff
@@ -8,6 +8,7 @@ import { firebaseTimestampToMoment } from '../utils/date'
 import DiagnoseImages from '../DiagnoseImages'
 import classes from '../stylesheets/admin.css'
 import { getUserUIDFromDiagnoseRef } from '../utils/diagnose'
+import _ from 'lodash'
```

Comment:
Please import only what you need from a package: `import { isEmpty } from 'lodash'`